docs/UserGuide.adoc

```diff
@@ -200,28 +200,28 @@ along with their index which can be used to display actual consultation details.
 
 === Medicine Management
 [[addMed, Add medicine]]
-==== Add medicine into storage: `addMed`
+==== Add medicine into storage: `addmed`
 
 Add a new medicine under directory specified by the path. +
 
-Format: `addMed PATH MEDICINE_NAME [QUANTITY]`
+Format: `addmed PATH MEDICINE_NAME [QUANTITY]`
 
 Example:
 
-* `addMed root\TCM healroot 50` +
-Add a new medicine with name "healroot" and quantity 50 at directory root\TCM
+* `addmed root\cough ibuprofen 50` +
```

Comment:
Implementation of addmed is changed to accommodate pricing. I will change the UG myself

Reply:
yea will discuss this tomorrow during the meeting.